src/main/webapp/index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>Piece of Cake!</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
     <script src="script.js"></script>
   </head>
   <body>
-        <h3>Welcome to Piece of Cake!</h3>
+    <div id="header" class="wrapper">
```

Comment:
In general, avoid adding `id`s to target elements. Instead, use a className. You can add multiple classNames by seperating them by a space `class="wrapper header"`. An ID can only be used once throughout your document, so this can lead to conflicts.